src/main/java/core/gui/comp/entry/DoppelLabelEntry.java

```diff
@@ -33,7 +38,7 @@ public DoppelLabelEntry(Color color) {
         super();
         m_clLinks = new ColorLabelEntry("", ColorLabelEntry.FG_STANDARD, color,
                                         SwingConstants.RIGHT);
-        m_clRechts = new ColorLabelEntry("", core.model.UserParameter.instance().FG_INJURED, color,
+        m_clRechts = new ColorLabelEntry("", DIFF_COLOR, color,
```

Comment:
I am sorry if it is a dumb remark , but as a first look it seems this should not be required if we drop coloring player name ... I guess I am missing something

Reply:
It is the colour of the delta for the different skills of the player when downloading data from HT.